tests/test_models.py

```diff
@@ -630,3 +630,16 @@ class TestProperties(
     assert "gsd" in properties
     assert "view:off_nadir" in properties
     assert "landsat:path" in properties
+
+
+def test_extension():
+    test_item = request(EO_EXTENSION)
+    test_item["stac_extensions"].append("foo")
+
+    item = Item.parse_obj(test_item)
+    assert "foo" in item.stac_extensions
+
+    test_collection = request(COLLECTION)
```

Comment:
really minor but should this be separated into different tests for debugging purposes?